Citadel/Saurfang.lua

```diff
@@ -118,6 +120,7 @@ function mod:Mark(args)
 end
 
 function mod:Frenzy(args)
+	self:SetStage(2)
```

Comment:
Do you really want to change stage for frenzy? do the timers change? Usually a SetStage really means the fight changes - not just when a boss does more damage.

Reply:
And keep in mind, players can just do a Boss > X health check for these things (or active buff)

Reply:
Probably not. I just tried to think of what "could" be considered a new phase. Will remove.